.spraygun.js

```diff
@@ -14,10 +14,11 @@ exports.setup = (projectDirectory, { chalk, shell }) => {
     [/app-prototype/g, appName, "Dockerfile"],
     [/app-prototype/g, appName, "README.md"],
     [/app-prototype/g, appName, "package.json"],
-    [/"node": "\d+\.\d+\.\d+"/, `"node": "${nodeVersion}"`, "package.json"],
+    [/"node": "\^\d+\.\d+\.\d+"/, `"node": "${nodeVersion}"`, "package.json"],
     [/node:\d+\.\d+\.\d+/, `node:${nodeVersion}`, "Dockerfile"],
     [/nodejs \d+\.\d+\.\d+/, `nodejs ${nodeVersion}`, ".tool-versions"],
     [/node:\d+\.\d+\.\d+/, `node:${nodeVersion}`, ".circleci/config.yml"],
+    [/tag: "\d+\.\d+\.\d+"/, `tag: "${nodeVersion}"`, ".circleci/config.yml"],
```

Comment:
I tweaked these regexes to ensure that the node version is replaced in the spraygun-generated copy of this repo. Specifically there are node versions in the CircleCI config in the form `tag: "<VER>"` that weren't getting substituted.

This means that if someone in the future is running, say, node 12.13.2, the generated project will automatically say "12.13.2" everywhere.